generator/docs/GenerationTypes.md

```diff
@@ -26,8 +26,8 @@ The values that are generated by the generator are shown in the matrix below. `g
 | Data type | where there are no constraints for the field <br /> _(or only `ofType` and/or `not(is null)`)_ | where constraints exist for the field <br /> _(beyond `ofType` and `not(is null)`)_ |
 | ---- | ---- | ---- |
 | **string** | "Lorem Ipsum" | a string as short as possible given the intersection of other constraints <br /> a string as long as possible given the intersection of other constraints <br /> If a containsRegex or matchingRegex is present, use this to inform the characters that are included in the shortest/longest strings. Otherwise repeat a `<space>` character for the contents of both strings <br /> |
-| **numeric (integer values)** <br /> where all present numeric constraints (if any) contain whole numbers and no decimal points | -2147483648 <br /> 0 <br /> 2147483646 | _A unique set of:_ <br /> the lowest possible value or -2147483648 <br /> 0 (if permitted) <br /> the highest possible value or 2147483646 |
-| **numeric (decimal values)** <br /> where some numeric constraints contain a decimal point (even if the fraction is 0, e.g. 10.0) | -2147483648 <br /> 0 <br /> 2147483646 | _A unique set of:_ <br /> the 2 lowest possible values or **??** [see #529](https://github.com/ScottLogic/datahelix/issues/529) <br /> 0 (if permitted) <br /> the 2 highest possible values or 1.797693134862315699E+308 & 1.797693134862315700E+308 |
+| **numeric (integer values)** <br /> where all present numeric constraints (if any) contain whole numbers and no decimal points | -100000000000000000000 <br /> 0 <br /> 100000000000000000000 | _A unique set of:_ <br /> the lowest possible value or -100000000000000000000 <br /> 0 (if permitted) <br /> the highest possible value or 100000000000000000000 |
+| **numeric (decimal values)** <br /> where some numeric constraints contain a decimal point (even if the fraction is 0, e.g. 10.0) | -100000000000000000000 <br /> 0 <br /> 100000000000000000000 | _A unique set of:_ <br /> the 2 lowest possible values or -100000000000000000000, -99999999999999999999 [see #529](https://github.com/ScottLogic/datahelix/issues/529) <br /> 0 (if permitted) <br /> the 2 highest possible values or 99999999999999999999, 100000000000000000000 |
```

Comment:
I think #529 is now solved? Please can you confirm this @sl-slaing

Reply:
Confirmed to be fixed, link removed